WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorView.swift

```diff
@@ -154,6 +159,21 @@ struct NewTaxRateSelectorView: View {
             .padding(.top, Layout.editTaxRatesInWpAdminSectionVerticalSpacing)
         }
     }
+
+    var resultsFixedBottomPanel: some View {
```

Comment:
I find `resultsFixedBottomPanel` and `listFooter` a bit hard to read, how about something like `taxRateFixedBottomSheetView`,  and `taxRateListFooterView` or something similar?

We can also make them private!

Reply:
Good hint, better naming and privatization in 9b55a10c8e168890d5d70fe355c0069e16c023d6